eng/mgmt/mgmtmetadata/compute_resource-manager.txt

```diff
@@ -3,12 +3,12 @@ AutoRest installed successfully.
 Commencing code generation
 Generating CSharp code
 Executing AutoRest command
-cmd.exe /c autorest.cmd https://github.com/Azure/azure-rest-api-specs/blob/master/specification/compute/resource-manager/readme.md --csharp --version=latest --reflect-api-versions --csharp-sdks-folder=C:\GH\azure-sdk-for-net\sdk
-2019-11-27 22:36:36 UTC
+cmd.exe /c autorest.cmd https://github.com/hapandar/azure-rest-api-specs/blob/dev/hapandar/AutoRepairsSwaggerUpdate/specification/compute/resource-manager/readme.md --csharp --version=latest --reflect-api-versions --csharp-sdks-folder=C:\repos\azure-sdk-for-net\azure-sdk-for-net\sdk
+2019-12-09 22:42:40 UTC
 Azure-rest-api-specs repository information
-GitHub fork: Azure
-Branch:      master
-Commit:      f57417493691c0d9a15440ffd7fdb4173bf7295c
+GitHub fork: hapandar
```

Comment:
I generated this from my branch since this code is not checked in yet. Should we generate this from master version once swagger changes are checked in or is this okay?